### PR TITLE
remove the positive blood culture model as this is no longer used #537

### PR DESCRIPTION
**Pull request by fredkingham**

### CODE REVIEW

File: apps/tb/migrations/0035_auto_20181018_0940.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (davidmiller) [inline apps/tb/migrations/0035_auto_20181018_0940.py:1]: This doesn't seem to be part of this PR?

File: elcid/migrations/0026_auto_20181018_0940.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        ),
+        migrations.RemoveField(
+            model_name='positivebloodculturehistory',
+            name='patient',
+        ),
+        migrations.RemoveField(
+            model_name='positivebloodculturehistory',
+            name='updated_by',
+        ),
+        migrations.AlterModelOptions(
```
Comment (davidmiller) [inline elcid/migrations/0026_auto_20181018_0940.py:27]: Why are all the below changing here?

File: elcid/test/test_api.py [hunk 3]
```diff
@@ -326,3 +326,30 @@ def test_patient_found_with_hash(self):
         self.assertEqual(
             response["patient"]["demographics"][0]["first_name"], "Dot"
         )
+
+
+class GetReferenceRangeTestCase(OpalTestCase):
```
Comment (davidmiller) [inline elcid/test/test_api.py:331]: Just for test coverage?

**Comment by davidmiller** [comment]
I think there's more code to delete:

```
ack --ignore-dir htmlcov positive_blood_culture_history
elcid/templates/partials/card_without_results.html
16:                  <span class="pull-right" ng-repeat="episode in [row]" ng-show="currentTag !== 'bacteraemia' && row.positive_blood_culture_history.length">

elcid/templates/partials/card_with_results.html
16:                  <span class="pull-right" ng-repeat="episode in [row]" ng-show="currentTag !== 'bacteraemia' && row.positive_blood_culture_history.length">

elcid/templates/partials/historical_bacteraemia.html
2:<span ng-show="episode.positive_blood_culture_history[0].when.add(6, 'months') | future">

intrahospital_api/templates/reconcile_list.html
51:                        <span class="pull-right" ng-repeat="episode in [row]" ng-show="currentTag !== 'bacteraemia' && row.positive_blood_culture_history.length">
```

**Comment by fredkingham** [comment]
this has been superseded by https://github.com/openhealthcare/elcid-rfh/pull/556
